tests/all_bad_test.py

```diff
@@ -48,16 +48,16 @@ class TestAllBad(object):
         # "f(,",
         # "f(x,,y)",
         # "f(x,y,",
-        "\\sin^x",
-        "\\cos^2",
+        "\\sin ^x",
```

Comment:
@builtbywill I remember you saying something about spaces the other day but I didn't quite understand. Why are we adding them now?

Reply:
@lily514 the main changes in here are to make things work with the spaces that will be automatically added by mathlive, which we caught when `\angle \frac{\pi }{3}` failed to parse, and would be the output of mathlive

I just went thru and tweaked some of the examples to more closely match the latex the app will actually be generating and sending out